src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -118,12 +122,15 @@ function SciMLBase.symbolic_discretize(pdesys::ModelingToolkit.PDESystem,discret
         # TODO: Generalize central difference handling to allow higher even order derivatives
         # The central neighbour indices should add the stencil to II, unless II is too close
         # to an edge in which case we need to shift away from the edge
+
         # Calculate buffers
-        I1 = oneunit(first(indices))
-        Imin = first(indices) + I1 * (order÷2)
-        Imax = last(indices) - I1 * (order÷2)
+        # TODO: check central_neighbor_idxs. It does not work like the old implementation.
```

Comment:
hmm, weird. I don't mind taking a look at this again after it's merged.

Reply:
Thanks!